python/lsst/sims/catUtils/utils/ObservationMetaDataGenerator.py

```diff
@@ -314,27 +207,27 @@ def getObservationMetaDataOld(self, obsHistID=None, expDate=None, fieldRA=None,
 
                 if isinstance(value, tuple):
                     if len(value) > 2:
-                        raise RuntimeError('Cannot pass a tuple longer than 2 elements '+
+                        raise RuntimeError('Cannot pass a tuple longer than 2 elements ' +
                                            'to getObservationMetaData: %s is len %d'
-                                           % (column[0], len(value)))
+                                           % (column, len(value)))
 
                     # perform any necessary coordinate transformations
-                    if column[4] is not None:
-                        vmin = column[4](value[0])
-                        vmax = column[4](value[1])
+                    if transform[1] is not None:
+                        vmin = transform[1](value[0])
+                        vmax = transform[1](value[1])
                     else:
                         vmin = value[0]
                         vmax = value[1]
 
                     query += ' %s > %s AND %s < %s' % \
-                             (column[1], vmin, column[1], vmax)
+                             (transform[0], vmin, transform[0], vmax)
                 else:
                     # perform any necessary coordinate transformations
-                    if column[4] is not None:
-                        vv = column[4](value)
+                    if transform[1] is not None:
+                        vv = transform[1](value)
                     else:
                         vv = value
-                    query += ' %s == %s' % (column[1], vv)
+                    query += ' %s == %s' % (transform[0], vv)
 
                 nConstraints += 1
 
```

Comment:
Why are observations being grouped by expMJD ? In OpSim outputs, there are no two observations with the same expMJD (once you purge of duplicates). In principle, (say if you have a beam splitter), one might have two observations with the same expMJD. Will this still work for that case? 

Is this basically trying to order observations? If so can we use ` ORDER BY expMJD`?